node/modules/mpoolnonceapi.go

```diff
@@ -21,13 +22,41 @@ type MpoolNonceAPI struct {
 	StateAPI full.StateAPI
 }
 
-// GetNonce gets the nonce from actor state
+// GetNonce gets the nonce from current chain head.
 func (a *MpoolNonceAPI) GetNonce(addr address.Address) (uint64, error) {
```

Comment:
Is this code correct? If so, we may want to do this in the mpool itself to avoid computing tipsets unnecessarily.

Reply:
or should there be a +1 here?